src/openrct2-ui/windows/Sign.cpp

```diff
@@ -178,8 +178,8 @@ rct_window * window_sign_open(rct_windownumber number)
     sint32 view_z = map_element->base_height << 3;
     w->frame_no = view_z;
 
-    w->list_information_type = map_element->properties.scenerymultiple.colour[0] & 0x1F;
-    w->var_492 = map_element->properties.scenerymultiple.colour[1] & 0x1F;
+    w->list_information_type = BASE_COLOUR(map_element->properties.scenerymultiple.colour[0]);
```

Comment:
I understand why you did this, but this is not what BASE_COLOUR is intended for.

Please create a define (not an enum, a define) called SCENERY_LARGE_COLOUR_MASK, with a value of 0x1F.